inc/class_error.php

```diff
@@ -203,7 +203,7 @@ function error($type, $message, $file=null, $line=0, $allow_output=true)
 			$this->email_error($type, $message, $file, $line);
 		}
 
-		if($allow_output === true)
+		if($allow_output === true && $mybb->settings['errortypemedium'] != "none")
```

Comment:
This may prevent MyBB's error page from showing - instead we could change https://github.com/mybb/mybb/blob/2da5152718ad740bf4d2648829c856a5035a9a01/inc/class_error.php#L221 to
```php
elseif(in_array($mybb->settings['errortypemedium'], array('warning', 'both')))
```
so that all details are printed when _Error Type Medium_ is set either of these options.

Reply:
Not if _errors_ imply _error details_ (the information being logged) - otherwise a blank page would be shown.

Reply:
Perhaps we could update the setting description from `Hide Errors and Warnings` to `Hide Errors and Warnings Details` then ?

Reply:
Maybe
```diff
- The type of errors to show.
+ The type of errors to show with detailed information.
```